src/index.ts

```diff
@@ -28,17 +38,19 @@ export const persist = (name, store, options = {}) => {
   })
 
   return storage.getItem(name)
-    .then((data) => {
+    .then((data: any) => {
```

Comment:
I was thinking of making it `object | string` but that causes some issues and may not handle everything properly (will any other adaptor return something other than those two though? I don't think so... and `snapshot` does have to be of `IStateTreeNode` type I'm pretty sure which I'm pretty sure extends object.

Reply:
This is an internal type that's (not yet) exposed in the typings (it might be if we expose a Storage interface), so shouldn't worry about handling all cases, just the ones here.